pkg/stores.go

```diff
@@ -0,0 +1,37 @@
+package ocr2keepers
+
+import "fmt"
+
+type ResultStore[T any] interface {
```

Comment:
Let's support multiple items per operation for `Add(...T)` and `Remove(...T)`

Reply:
Let's add this to the follow-up card too so it doesn't block forward momentum

Reply:
I'm good with var args.